submissions/dikun/js-dom/script.js

```diff
@@ -0,0 +1,86 @@
+        },
+        chersonesos: {
+            id: "chersonesos",
+            title: 'chersonesos',        
+            img: 'img/khersones.jpg',
+            descr: "Chersonesos was an ancient Greek colony founded approximately 2500 years ago in the southwestern part of Crimea, known then as Taurica. The ancient city is located on the shore of the Black Sea at the outskirts of Sevastopol and has been nicknamed the 'Ukrainian Pompeii'. Chersonesos' ancient ruins are presently located in one of Sevastopol's suburbs. They are today a popular tourist attraction, protected by the state as an archaeological park. The buildings mix influences of Greek, Roman and Byzantine culture. The defensive wall is hundreds of meters long. Buildings include Roman amphitheatre and a Greek temple."
+        },
+        sophia: {
+            id: "sophia",
+            title: 'saint sophia cathedral',       
+            img: 'img/sophia.webp',
+            descr: "Chersonesos was an ancient Greek colony founded approximately 2500 years ago in the southwestern part of Crimea, known then as Taurica. The ancient city is located on the shore of the Black Sea at the outskirts of Sevastopol and has been nicknamed the 'Ukrainian Pompeii'. Chersonesos' ancient ruins are presently located in one of Sevastopol's suburbs. They are today a popular tourist attraction, protected by the state as an archaeological park. The buildings mix influences of Greek, Roman and Byzantine culture. The defensive wall is hundreds of meters long. Buildings include Roman amphitheatre and a Greek temple."
+        },
+        khotyn: {
+            id: "khotyn",
+            title: 'the khotyn fortress',        
+            img: 'img/khotyn.jpg',
+            descr: "The Khotyn Fortress is a fortification complex located on the shores of the Dniester River in Khotyn in western Ukraine. The construction of the current fortress was started in 1325. The fortress is a large tourist attraction in the area and Ukraine. Thanks to its preserved historical atmosphere the fortress is often used as a setting for movies. It has represented various French and English castles, historical locations, and fortifications. Most recently, it appeared in a Russian movie Taras Bulba, The Three Musketeers, or the Arrows of Robin Hood."
+        }
+    };
+
+    const list = document.querySelector(".list_items");
+    const content = document.querySelector(".content");
+    
+    function createListHtml(allMyWonders) {
+        Object.values(allMyWonders).forEach((key) => {
+            list.innerHTML += `<li class="list_item">
```

Comment:
Common mistake - adding DOM elements in loop https://kottans.org/documentation/docs/doc/code-review/#dom-api